packages/x-date-pickers/src/ClockPicker/ClockPicker.tsx

```diff
@@ -63,20 +67,9 @@ export interface ExportedClockPickerProps<TDate> extends ExportedTimeValidationP
   ) => string;
 }
 
-export interface ClockPickerSlotsComponent {
-  LeftArrowButton: React.ElementType;
-  LeftArrowIcon: React.ElementType;
-  RightArrowButton: React.ElementType;
-  RightArrowIcon: React.ElementType;
-}
-
-// We keep the interface to allow module augmentation
-export interface ClockPickerComponentsPropsOverrides {}
+export interface ClockPickerSlotsComponent extends PickersArrowSwitcherSlotsComponent {}
```

Comment:
The slots were duplicated, `ClockPicker` should not have been responsible of this abstraction

Reply:
What do you mean by this comment?
I don't see that anything would have been changed here. 🤔

Reply:
Oh @alexfauquette already did the fix on #5893 so my comment does not make sense anymore
Basically we add the slots of the `PickersArrowSwitcher` redefined here instead of extending the interface.